dev/stats/get_important_pr_candidates.py

```diff
@@ -45,73 +46,180 @@
     required=True,
     help=textwrap.dedent(
         """
-        GitHub token used to authenticate.
-        You can set omit it if you have GITHUB_TOKEN env variable set
+        A GitHub token is required, and can also be provided by setting the GITHUB_TOKEN env variable.
         Can be generated with:
         https://github.com/settings/tokens/new?description=Read%20issues&scopes=repo:status"""
     ),
     envvar='GITHUB_TOKEN',
 )
 
-PROVIDER_SCORE = 0.5
-REGULAR_SCORE = 1.0
 
-REVIEW_INTERACTION_VALUE = 1.0
```

Comment:
Moving these inside the class lets us import the class into a notebook, needed for successful unpickling.